react/src/redux/books/actions.js

```diff
@@ -90,6 +90,29 @@ const actionCreators = {
         }
       }
     }
+  },
+  getCommentaries: (bookId) => {
+    return async (dispatch) => {
+      dispatch({type: actionTypes.BOOK_LOADING});
+      const responseCommentaries = await BookService.getBookCommentaries(bookId);
+      if(responseCommentaries.ok){
+        dispatch({type: actionTypes.GET_BOOK_COMMENTARIES, payload: {commentaries: responseCommentaries.data}});
+      }else{
+        dispatch({type: actionTypes.GET_BOOK_COMMENTARIES, payload: {commentaries: null}});
+      }
+    }
+  },
+  postComment: (bookId, comment) => {
+    return async (dispatch) => {
+      dispatch({type: actionTypes.BOOK_LOADING});
+      const responseData = await BookService.getUserData();
+      if(responseData.ok){
+        const responsePost = await BookService.postBookComment(bookId, responseData.data.id, comment);
+        if(responsePost.ok){
+          actionCreators.getCommentaries(bookId)(dispatch)
```

Comment:
`dispatch(actionCreators.getCommentaries(bookId))` ? Remember that the middleware provides `dispatch` and `getState` functions for every function passed to it.